ci-operator/config/integr8ly/grafana-operator/integr8ly-grafana-operator-master.yaml

```diff
@@ -0,0 +1,50 @@
+base_images:
+  os:
+    name: alpine
+    namespace: alpine
+    tag: "3.8"
```

Comment:
Any reason to use this base image? We use ubi for everything else

Reply:
No real reason other than the operator being a upstream focused project. But UBI would work for that too I guess. Would you recommend to change the base image @mikenairn ?